pkg/network/ebpf/c/protocols/protocol-classification-helpers.h

```diff
@@ -24,6 +27,13 @@ long bpf_skb_load_bytes_with_telemetry(const void *skb, u32 offset, void *to, u3
         return false;                                                       \
     }                                                                       \
 
+// Kernels before 4.7 do not know about per-cpu array maps.
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
```

Comment:
given the usermode change that we have just made for the prebuilt, do we still need that ifdef?

Reply:
Yes, because of runtime-compilation on kernel <4.7.